### PR TITLE
token wrapper handler via sig bridge tests

### PR DESCRIPTION
**Pull request by saiakilesh**

### CODE REVIEW

File: pallets/signature-bridge/src/tests.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ fn setup_resources() {
 
 		assert_ok!(Bridge::set_resource(Origin::root(), id, method.clone()));
 		assert_eq!(Bridge::resources(id), Some(method));
-
+
```
Comment (drewstone) [inline pallets/signature-bridge/src/tests.rs:45]: Extra space should be removed.

File: pallets/token-wrapper-handler/src/tests_signature_bridge.rs [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+		r_id: *resource_id,
+		name,
+		asset_id,
+	})
+}
+
+// ----Signature Bridge Tests----
+
+#[test]
+
```
Comment (drewstone) [inline pallets/token-wrapper-handler/src/tests_signature_bridge.rs:65]: Extra line should be removed.

**Review by drewstone — CHANGES_REQUESTED** [review]
Looks great!

Seems there are updates to
- fixtures submodule
- orml

Make sure you've committed what changes you may have in those sub modules or check out the right ones and commit to your repo. What changed within those by the way?

File: pallets/token-wrapper-handler/src/tests_signature_bridge.rs (hunk 2 above)
Comment (saiakilesh) [reply pallets/token-wrapper-handler/src/tests_signature_bridge.rs]: fixed

**Comment by saiakilesh** [comment]
> Looks great!
> 
> Seems there are updates to
> 
> * fixtures submodule
> * orml
> 
> Make sure you've committed what changes you may have in those sub modules or check out the right ones and commit to your repo. What changed within those by the way?

I think for some reason when I merged main, the orml submodule stayed on an old commit so I had to manually checkout the right commit, as you wrote. I didn't end up changing anything within those submodules.

**Pull request merged**